python/gtsam/tests/test_Triangulation.py

```diff
@@ -15,21 +15,24 @@
 import gtsam as g
 from gtsam.utils.test_case import GtsamTestCase
 from gtsam import Cal3_S2, Cal3Bundler, Rot3, Pose3, \
-    PinholeCameraCal3_S2, Point3, Point2Vector, Pose3Vector, triangulatePoint3
+    PinholeCameraCal3_S2, Point3, Point2Vector, Pose3Vector, triangulatePoint3, CameraSetCal3_S2, CameraSetCal3Bundler, PinholeCameraCal3Bundler
 
 class TestVisualISAMExample(GtsamTestCase):
-    def test_TriangulationExample(self):
-        # Some common constants
-        sharedCal = Cal3_S2(1500, 1200, 0, 640, 480)
-
+    def setUp(self):
+        # Set up two camera poses
         # Looking along X-axis, 1 meter above ground plane (x-y)
         upright = Rot3.Ypr(-np.pi / 2, 0., -np.pi / 2)
-        pose1 = Pose3(upright, Point3(0, 0, 1))
-        camera1 = PinholeCameraCal3_S2(pose1, sharedCal)
+        self.pose1 = Pose3(upright, Point3(0, 0, 1))
 
         # create second camera 1 meter to the right of first camera
-        pose2 = pose1.compose(Pose3(Rot3(), Point3(1, 0, 0)))
-        camera2 = PinholeCameraCal3_S2(pose2, sharedCal)
+        self.pose2 = self.pose1.compose(Pose3(Rot3(), Point3(1, 0, 0)))
```

Comment:
Looks like you dont explicitly need to make them members. i.e maybe you declare pose1 and pose2 in setUp as they were done before and use them in the other tests?

Reply:
My bad, did not read the diff correctly. You may ignore this, marking as resolved :/